cypress/integration/home.spec.ts

```diff
@@ -1,4 +1,48 @@
-it('should redirect to home page', () => {
-  cy.visit('/');
-  cy.location('pathname').should('eq', '/');
-});
+const getTaskTitleInput = () => cy.get('[data-testid="task-title-input"]');
+const getAddTaskButton = () => cy.get('[data-testid="add-task-button"]');
+const getTitleMsgError = () => cy.get('[data-testid="title-msg-error"]');
+const getModalActionTask = () => cy.get('[data-testid="modal-msg-action-task"]');
+const getButtonCheckedTask = () => cy.get('[data-testid="button-checked-task"]');
+const getTextTitleTask = () => cy.get('[data-testid="text-title-task"]');
+const getButtonRemoveTask = () => cy.get('[data-testid="button-remove-task"]');
+
+it('should appear title error message when title input is empty', () => {
+    cy.visit('/');
+    getTaskTitleInput().click();
+    getAddTaskButton().click();
+    getTitleMsgError().should('be.visible');
+})
+
+it('Should throw an error when the input is less than five characters', () => {
+    cy.visit('/');
+    getTaskTitleInput().type('1234');
+    getAddTaskButton().click();
+    getTitleMsgError().should('be.visible');
+})
+
+it('Should generate an error when the input has more than ten characters', () => {
+    cy.visit('/');
```

Comment:
I suggest using `beforeEach` to avoid `cy.visit('/');` repetition.

```ts
beforeEach(() => {
  cy.visit('/');
});
```

Reply:
Resolved on cf4041d82f46b410d9200b943ed74dae9c181a96